.scripts/create_push_client.sh

```diff
@@ -17,48 +17,49 @@ function generate_client_and_create_pr() {
     local message="chore: update client version to ${newVersion}"
     local body=$(pr_body)
     local short_head=$(git rev-parse --short HEAD)
-    local branch="client_update_${short_head}"
+    local branch="client_update_${short_head}_$(date +%s)"
 
     cd /tmp/${GHREPO}
     git checkout -b ${branch}
     cd -
+    for i in $(find tool -name "*.go"); do
+        sed -i 's:"github.com/fabric8-services/fabric8-cluster/cluster":"github.com/fabric8-services/fabric8-cluster-client/cluster":' "$i";
```

Comment:
There should not be any repo specific code (fabric8-cluster-*) in common. The code in common should work for any repo or it should be moved to fabric8-cluster-client repo.

Reply:
Addressed in a39b4f2